README.md

```diff
@@ -197,6 +197,29 @@ const program = createProgram(
 program.emit(undefined, undefined, undefined, undefined, di({ program }));
 ```
 
+### Usage with ttypescript
+
+To use DI-compiler With [`ttypescript`](https://github.com/cevek/ttypescript), you need to write a transformer and set the path to `tsconfig.json`.
+
+```typescript:transformer.ts
+import type { Program } from 'typescript'
+import { di } from "@wessberg/di-compiler";
+
+const transformer = (program: Program) => di({ program })
+
+export default transformer
+```
+
+```tsconfig.json
+{
+    "compilerOptions": {
+        "plugins": [
+            { "transform": "path/to/transformer.ts" },
```

Comment:
```suggestion
            { "di": "path/to/my-di-transformer.ts" },
```

Reply:
It doesn't seem to work. `ttypescript`(@1.5.12) requires `"transform"` as key.

```
Error: tsconfig.json plugins error: transform must be present
```

Reply:
> It doesn't seem to work. `ttypescript`(@1.5.12) requires `"transform"` as key.
> 
> ```
> Error: tsconfig.json plugins error: transform must be present
> ```

Ah, OK, I didn't know that. I'll remove the suggestion